packages/core/contracts/financial-templates/funding-rate-store/implementation/FundingRateStore.sol

```diff
@@ -270,7 +310,10 @@ contract FundingRateStore is FundingRateStoreInterface, Testable, Lockable {
         );
 
         // Compute proposal bond.
-        FixedPoint.Unsigned memory proposalBond = proposalBondPct.mul(AdministrateeInterface(perpetual).pfc());
+        RecordParamsForContract storage recordParam = recordParams[perpetual];
+        FixedPoint.Unsigned memory proposalBond = recordParam.current.proposerBondPct.mul(
+            AdministrateeInterface(perpetual).pfc()
```

Comment:
do you want to add an accessor method to read the `pfc` from a given `perpetual`?

Reply:
like an internal `getPfc(perpetual)` method?

Reply:
yes

Reply:
Made the change because it improves readability, small hit to bytecode (<100 bytes)